### PR TITLE
445 Main and AtRule to use new PostCSS Syntax

### PR DESCRIPTION
**Pull request by peterramsing**
**What kind of change is this? (Bug Fix, Feature...)**

**What is the current behavior (You can also link to an issue)**

**What is the new behavior this introduces (if any)**

**Does this introduce any breaking changes?**

**Does the PR fulfill these requirements?**
- [x] Tests for the changes have been added
- [x] Docs have been added or updated

**Other Comments**

### CODE REVIEW

File: lib/lost-at-rule.js [hunk 1]
```diff
@@ -1,36 +1,32 @@
-module.exports = function lostAtRule(css, Settings) {
-  css.walkAtRules('lost', function lostAtRuleFunction(Rule) {
-    var rule = Rule;
-    var settings = Settings;
+module.exports = function lostAtRule(rule, settings) {
+  const breakoutPramams = rule.params.split(' ');
 
-    rule.params = rule.params.split(' ');
+  if (breakoutPramams[0] === 'gutter') {
```
Comment (peterramsing) [inline lib/lost-at-rule.js]: Use a `case` here?

**Comment by ThornWalli** [comment]
@peterramsing What is the current status here? The priority of postcss update becomes more and more importance...

Get this message when using this plugin, update of `postcss-preset-env` results in incompatibility.

image

**Comment by peterramsing** [comment]
Look into getting this releases asap.

**Comment by peterramsing** [comment]
All tests are passing and other issues seem to be resolved with this PR. I'm not ready to release a non-beta version of v8 but this PR seems buttoned up.

**Pull request merged**